docs/plugins/firebase.md

```diff
@@ -265,84 +263,32 @@ The prior example requires the `embedding` field to be indexed to work. To creat
 
 ### Deploy flows as Cloud Functions
 
-The plugin provides the `onFlow()` constructor, which creates a flow backed by a Cloud Functions for Firebase HTTPS-triggered function. These functions conform to Firebase's [callable function interface](https://firebase.google.com/docs/functions/callable-reference) and you can use the [Cloud Functions client SDKs](https://firebase.google.com/docs/functions/callable?gen=2nd#call_the_function) to call them.
+To deploy a flow with Cloud Functions, use the Firebase Functions library's native support for genkit. The `onCallGenkit` method allows
+you to create a [callable function](https://firebase.google.com/docs/functions/callable?gen=2nd) from a flow. It will automatically support
+streaming and JSON requests. You can use the [Cloud Functions client SDKs](https://firebase.google.com/docs/functions/callable?gen=2nd#call_the_function) to call them.
 
 <!--See note above on prettier-ignore -->
 <!-- prettier-ignore -->
 ```js
-import { onFlow, noAuth } from "@genkit-ai/firebase/functions";
+import { onCallGenkit } from 'firebase-functions/https';
+import { defineSecret } from 'firebase-functions/params';
 
-export const exampleFlow = onFlow(
-  ai, // Provide the Genkit instance
-  {
-    name: "exampleFlow",
-    authPolicy: noAuth(), // WARNING: noAuth() creates an open endpoint!
-  },
-  async (prompt) => {
+export const exampleFlow = ai.defineFlow({
+  name: "exampleFlow",
+}, async (prompt) => {
     // Flow logic goes here.
 
     return response;
   }
 );
+
+// WARNING: This has no authentication or app check protections. See Cloud Functions for Firebase
```

Comment:
Point to https://github.com/firebase/genkit/pull/1798/files#diff-d9cbd97d17d60b9a415f168c988e7af55ab4d6945e24b0fb570a0577eff6b884R112 for now?

Reply:
Added a link to GH docs for now.